pos_multi_session/pos_multi_session_models.py

```
@@ -42,23 +42,39 @@ def on_update_message(self, message):
         return res
 
     @api.multi
+    def order_number_check(self, message):
+        self.ensure_one()
+        order_uid = message['data']['uid']
+        client_revision_ID = message['data']['revision_ID']
+        server_revision_ID = self.env['pos.multi_session.order'].search([('order_uid', '=', order_uid)]).revision_ID
+        if client_revision_ID is not server_revision_ID:
+            print "The client is not updated"
+            return False
```

I guess you need to raise an Exception here